buildkite/docker/Dockerfile

```diff
@@ -99,7 +99,15 @@ RUN apt-get -qqy update && \
 RUN curl -L https://deb.nodesource.com/setup_8.x | bash - && \
```

Comment:
NodeJS should be upgraded to 10.x also, see the one used in [Gerrit-CI](https://gerrit.googlesource.com/gerrit-ci-scripts/+/refs/heads/master/jenkins-docker/slave-node-wct/Makefile)

Reply:
Indeed, NodeJs is there as well for Debian: https://deb.nodesource.com/setup_10.x.

Reply:
Done.